testflinger_agent/client.py

```diff
@@ -55,6 +58,23 @@ def _requests_retry(self, retries=3):
         session.mount("https://", adapter)
         return session
 
+    def _configure_influx(self):
+        host = os.environ.get("INFLUX_HOST")
+        port = os.environ.get("INFLUX_PORT", 8086)
+        user = os.environ.get("INFLUX_USER")
+        password = os.environ.get("INFLUX_PW")
```

Comment:
What happens if these env vars are not configured? We need to account for that possibility and handle it gracefully so that the code later doesn't have to figure out whether it should log to influx or not. Otherwise we'll run into problems